microsoft-edge/webdriver-chromium/index.md

```diff
@@ -71,7 +71,7 @@ This article provides instructions for using the Selenium framework, but you can
 
 ### Using Selenium 4
 
-Selenium WebDriver is an open-source testing framework that can be used on any platform, and provides language bindings for Java, Python, C#, Ruby, and JavaScript.
+Selenium WebDriver is an open-source testing framework that can be used on any platform, and provides language bindings for Java, Python, C#, Ruby, and JavaScript. Note that for Python users, Python 3 is required to run Selenium 4 tests, Python 2.7 is not supported.
```

Comment:
Python 3 is only required _if_ you want to follow our Selenium 4 examples.

You can use a WebDriver client written in any language, _including Python 2.7_. You can use clients and languages as long as they are W3C-compliant.

Reply:
Note: Python 3 is required to run Selenium 4 tests.  (Python 2.7 is not supported.)

Reply:
This is just a recommendation to streamline the language.

Reply:
Thank you for the suggestions. I have updated the doc as suggested. Please merge the PR if this looks good. (I don't have permission to merge this PR to the repo.) @JasonAndrewWriter @mikehoffms